reconcile/typed_queries/saas_files.py

```diff
@@ -75,6 +76,13 @@ class Config:
         # ignore `namespaceSelector` and 'provider' fields from the GQL schema
         extra = Extra.ignore
 
+    def uid(self, saas_file_name: str, resource_template_name: str) -> str:
+        """Returns a unique identifier for a target."""
+        return hashlib.blake2s(
+            f"{saas_file_name}:{resource_template_name}:{self.name if self.name else 'default'}:{self.namespace.cluster.name}:{self.namespace.name}".encode(),
```

Comment:
should we use the saas-file `name` field instead of `saas_file_name`?

Reply:
I don't know what you mean. `saas_file_name` here is the `name` attribute of the saas-file. The context here (`self`) is the target.

Reply:
ok, i thought it was the actual file name... sorry!